src/main/java/twilightforest/world/TFGenCaveStalactite.java

```diff
@@ -24,6 +28,9 @@ public class TFGenCaveStalactite extends TFGenerator {
     public int maxLength;
     public int minHeight;
 
+    // Greg?
+    public boolean doOreGen;
+
```

Comment:
The `// Greg?` comment appears to be misleading since this can be used without GregTech. Also you should generally not expose fields publicly so this should be private.